statsmodels/emplike/tests/test_regression.py

```diff
@@ -13,11 +13,12 @@ class GenRes(object):
     Loads data and creates class instance ot be tested
 
     """
-    def __init__(self):
+    @classmethod
+    def setup_class(cls):
```

Comment:
are those changes, replacing `__init__` by `setup_class` required or just to save on repeated `__init__` computation>